files-community/Files
# Fix: Fixed an issue with launching programs from the Address Bar

**Resolved / Related Issues**
- Closes #15455 

**Steps used to test these changes**

1. Create `test1.bat` and `test 1.bat` both with the same contents as below and place it in `C:\Folder`,
    ```bat
    @echo off
    echo %*
    ```
2. Run each line from below through the address bar,
    ```
    // should fail
    C:\Folder\test 1
    C:\Folder\test 1 param1
    C:\Folder\test 1"
    "C:\Folder\test 1 param1
    "C:\Folder\test 1 param1"
    
    // should pass
    C:\Folder\test 1.bat
    "C:\Folder\test 1"
    "C:\Folder\test 1" param1
    "C:\Folder\test 1" param1 param2 "param 3"
    "C:\Folder\test 1
    C:\Folder\test1
    C:\Folder\test1 param1
    C:\Folder\test1 param1 param2 "param 3"
    "C:\Folder\test 1.bat"     param1           param2
    ```

**Undefined behavior**
The below work as well, although it's not clear how exactly these should be handled.
```
C:\Folder\test1""
"C:\Folder\test1"""" param1 param2
"C:\Folder\test 1""""" param1 param2 "param 3"
```
